example_rails6/Gemfile

```diff
@@ -48,3 +48,6 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 gem 'rolemodel_rails', path: '..'
+
+gem 'react-rails'
+gem 'slim'
```

Comment:
We don't want to commit the changes to the example Rails app as it is intended to be a base app with out any of our generators applied.